__tests__/html2/activityOrdering/partGroupingAtTheEndOfChatHistory.html

```diff
@@ -95,9 +95,9 @@
         });
 
         expect(pageElements.activityContents().map(({ textContent }) => textContent)).toEqual([
+          'a-00002: Hello, World at t = 1',
```

Comment:
The CoT unit is t = 0.

Hello, World is t = 1.

So "Hello, World" should appear after CoT.